Makefile

```diff
@@ -35,6 +35,10 @@ ifneq ($(VERSION), "")
 endif
 
 build-access:
+	# Build vue application(s) files
+	npm --prefix static/js/vue-cdr-access install
+	npm --prefix static/js/vue-cdr-access run build
```

Comment:
Will the tests always run? Not sure if that's a good or a bad thing, probably depends on how long they end up taking.

Reply:
Not in this setup. They should run on Travis or via the command line